lusSTR/tests/test_format.py

```diff
@@ -71,15 +72,28 @@ def test_format_sex_loci_straitrazor():
         assert filecmp.cmp(testformat, outfile_name_output) is True
 
 
-def test_uas_directory():
-    with NamedTemporaryFile() as outfile:
-        inputdb = data_file('UAS_bulk_input/')
-        testformat_auto = data_file('UAS_bulk_test.csv')
-        testformat_sex = data_file('UAS_bulk_test_sexloci.csv')
-        arglist = ['format', inputdb, '-o', outfile.name, '--uas', '--include-sex']
-        args = lusSTR.cli.get_parser().parse_args(arglist)
-        lusSTR.format.main(args)
-        assert filecmp.cmp(testformat_auto, outfile.name) is True
-        outfile_name = os.path.splitext(outfile.name)[0]
-        outfile_name_output = f'{outfile_name}_sexloci.csv'
-        assert filecmp.cmp(testformat_sex, outfile_name_output) is True
+def test_uas_directory_autosomal_only(tmp_path):
+    inputdb = data_file('UAS_bulk_input/')
+    copydb = str(tmp_path / 'UAS_bulk_input/')
+    copytree(inputdb, copydb)
+    with open(f'{copydb}/bogusfile.txt', 'w') as fh:
+        pass
```

Comment:
Doing this song and dance to verify that non `.xlsx` files in the input dir don't trip up the `format` command.